tests/integration/test_sqlalchemy_integration.py

```diff
@@ -0,0 +1,268 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+import pytest
+import sqlalchemy as sqla
+from sqlalchemy.sql import and_, or_, not_
+
+from trino.transaction import IsolationLevel
+
+
+@pytest.fixture
+def trino_connection(run_trino, request):
+    _, host, port = run_trino
+    engine = sqla.create_engine(f"trino://test@{host}:{port}/{request.param}",
+                                connect_args={"source": "test", "max_attempts": 1})
+    yield engine, engine.connect()
+
+
+@pytest.fixture
+def trino_connection_with_transaction(run_trino, request):
+    _, host, port = run_trino
+    engine = sqla.create_engine(f"trino://test@{host}:{port}/{request.param}",
+                                connect_args={"source": "test", "max_attempts": 1,
+                                              "isolation_level": IsolationLevel.READ_UNCOMMITTED})
```

Comment:
Does the isolation level have some significance in the tests? Asking to see if I missed something.

Reply:
This might be some trace from your PR as well. It doesn't have any significance here.